tpkg/credns.tdir/credns.test

```diff
@@ -0,0 +1,127 @@
+# #-- credns.test --#
+# source the master var file when it's there
+[ -f ../.tpkg.var.master ] && source ../.tpkg.var.master
+# use .tpkg.var.test for in test variable passing
+[ -f .tpkg.var.test ] && source .tpkg.var.test
+
+. ../common.sh
+
+export NSD_PATH="../../nsd"
+
+echo "** Starting servers"
+make start-slave
+wait_nsd_up slave/nsd.log
+make start-proxy
+wait_nsd_up proxy/nsd.log
+make start-master
+wait_nsd_up proxy/nsd.log
```

Comment:
Like one before, this needs to be master/nsd.log instead of the second proxy/nsd.log for wait_nsd_up.

Reply:
Done